src/LuaSystemBody.cpp

```diff
@@ -276,6 +276,28 @@ static int l_sbody_attr_apoapsis(lua_State *l)
 	return 1;
 }
 
+
+/*
+ * Attribute: orbitPeriod
+ *
+ * The orbit of the body, around its parent, in days
```

Comment:
Just so that the interface is clear, it might be a good thing to say whether the return value is an integer or not. I know it's not (or I've missed a step) but only by looking at the code, which isn't very user friendly.